frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/EnvironmentAccordionBody/EnvironmentAccordionBody.tsx

```diff
@@ -77,6 +78,13 @@ const StyledAlert = styled(Alert)(({ theme }) => ({
     marginInline: theme.spacing(2), // should consider finding a variable here
 }));
 
+const StyledStrategySeparator = styled(StrategySeparator)(({ theme }) => ({
+    [`&:has(+ *:not(ol) .${disabledStrategyClassName}), &:has(+ ol > li:first-of-type .${disabledStrategyClassName})`]:
+        {
+            filter: 'grayscale(1)',
```

Comment:
previously it was `grayscale` + `opacity` tweak

Reply:
Yes, but that was for the general contsraints (not the separator). I've removed the opacity tweak for now because it looked fine without it to my eyes and there's less of a risk of introducing contrast issues. If UX has any issues with it, we can always change it later